end_users/use/README.md

```diff
@@ -0,0 +1,30 @@
+# Use
```

Comment:
Calling this "use" is a bit confusing. Can we title something different? How about "workflow"?